src/boosteriferous/boosteriferous.csproj

```diff
@@ -27,14 +27,34 @@
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\lib\Assembly-CSharp.dll</HintPath>
-      <Private>False</Private>
+      <HintPath>R:\KSP_1.1.3_dev\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
yes, you are correct, i forgot about those

Problem with the src/lib was that they had 1.0 files in there, which 
messed me up until I realized what was going on.

JBB


On 8/25/2016 7:16 PM, Sound and Fury wrote:
>
> In src/boosteriferous/boosteriferous.csproj 
> <https://github.com/ec429/boosteriferous/pull/1#discussion_r76342080>:
>
> >      <Reference Include="Assembly-CSharp">
> > -      <HintPath>..\lib\Assembly-CSharp.dll</HintPath>
> > -      <Private>False</Private>
> > +      <HintPath>R:\KSP_1.1.3_dev\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
>
> Your personal reference paths aren't likely to be useful for anyone 
> else, especially when they're absolute paths starting with a drive 
> letter. (/Real/ filesystems have a global root, called |/|) That's 
> kinda why I did the src/lib/ thing in the first place, that way 
> there's at least a well-defined place for them to be.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ec429/boosteriferous/pull/1/files/a6d264e25a9fb3f362da2bc8ed6f385b08959fcc#r76342080>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABjf1gtiXQvVdJjM6k405VnieUEP5gs6ks5qjiI-gaJpZM4JtlwB>.
>